docs/source/private_data_tutorial.rst

```diff
@@ -731,16 +746,30 @@ When successful, should see something similar to the following result:
 Query private data Org2 is not authorized to
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Peers in Org2 do not have the Org1 asset1 ``appraisedValue`` private data in their side database.
-When they try to query for this data, they get back a hash of the key matching
-the public state but will not have the private state.
+Because the asset was created by Org1, peers in Org2 do not store the
```

Comment:
Because the asset was created by Org1, the ``appraisedValue`` associated with
``asset1`` is stored in the ``Org1MSPPrivateCollection`` collection. The value is
not stored by peers in Org2.